handsontable/package.json

```diff
@@ -41,6 +41,7 @@
     "test:e2e.verbose": "npm run build:umd && npm run build:languages && npm run test:e2e.dump && npm run test:e2e.puppeteer -- --verbose",
     "test:e2e.watch": "node ./test/scripts/run-puppeteer-on-watchers-change.mjs --cmdToListen \"npm run watch\" --cmdToListen \"npm run test:e2e.dump -- --watch\" --runnerFile \"test/E2ERunner.html\"",
     "test:e2e.dump": "cross-env-shell BABEL_ENV=commonjs_e2e NODE_ENV=test-e2e env-cmd -f ../hot.config.js webpack ./test/helpers/index.js ./test/e2e/index.js",
+    "test:e2e.dump.esm-cjs": "cross-env-shell BABEL_ENV=commonjs_e2e NODE_ENV=test-e2e-esm-cjs env-cmd -f ../hot.config.js webpack ./test/helpers/index.js ./test/e2e/index.js",
```

Comment:
I think this new script entry does the same as `test:e2e.dump` 🤔 What do you think about replacing it with `test:e2e.dump` in GH jobs? unless I missed something?

Reply:
it's using diff config `NODE_ENV=test-e2e-esm-cjs` which has a diff alias for building